src/models/FGFCS.cpp

```diff
@@ -129,8 +129,8 @@ bool FGFCS::InitModel(void)
   for (i=0; i<MixtureCmd.size(); i++) MixtureCmd[i] = 0.0;
   for (i=0; i<PropAdvance.size(); i++) PropAdvance[i] = 0.0;
   for (i=0; i<PropFeather.size(); i++) PropFeather[i] = 0.0;
-  for (i=0; i<OperationModeCmd.size(); i++) OperationModeCmd[i] = -1.0;
-  for (i=0; i<OperationMode.size(); i++) OperationMode[i] = -1.0;
+  for (i=0; i<OperationModeCmd.size(); i++) OperationModeCmd[i] = fdmex->GetPropertyValue("propulsion/engine/rocket/initial_state");
+  for (i=0; i<OperationMode.size(); i++) OperationMode[i] = fdmex->GetPropertyValue("propulsion/engine/rocket/initial_state");
```

Comment:
Better to not do this since it makes it very specific to a certain type of vehicle configuration file (i.e. if the property doesn't get created, JSBSim dies).

That being said, since this was a workaround to get the engine operation mode transition times working, all references to `OperationModeCmd` and `OperationMode` can be removed. Mind taking care of that? 🙂

Reply:
We can have a call so I can show you where the changes are, btw.